tests/ui/invalid_pyclass_enum.stderr

```diff
@@ -66,6 +66,14 @@ error: The `ord` option requires the `eq` option.
 83 | #[pyclass(ord)]
    |           ^^^
 
+error: All variants of enum `AllEnumVariantsDisabled` have been disabled by cfg attributes
```

Comment:
We should probably mention why it's not allowed:

```suggestion
error: #[pyclass] can't be used on enums without any variants - all variants of enum `AllEnumVariantsDisabled` have been configured out by cfg attributes
```

Reply:
Ok, good. I see we don't put #[pyclass] in backticks elsewhere in this module's compiler errors, too, so I haven't added them.